CHANGELOG.md

```diff
@@ -8,32 +8,86 @@ core section.***
 - ***When possible please provide the number of the pull request(s) containing the 
 changes in the following format: PR #1234***
 
-### LORIS 23.0 (Released: ??)
+## LORIS 23.0
 
+### Core
+#### Features
+- Add configuration option to specify the format in which dates are displayed. (PR #5004)
+- Configurable upload directory for data_release and document_repository modules (PR #5815)
+- New radio components for forms. (PR #5846)
 
-#### Core
+#### Updates and Improvements
+- PHP minimum requirements raised to PHP 7.3 (PR #5723)
 - Menus are now maintained by modules and no longer in the SQL database (PR #5839)
-- Very old instruments relying on QuickForm may have issues due to code changes (PR #4928)
-- Unix user permissions have been updated which may affect access to files. New
-documentation for file permissions has been added to the README.md file (PR #5323)
+- New script created for streamlining instantiation of the Raisinbread demonstration 
+database (PR #5260)
+- New documentation for file permissions has been added to the README.md file. (PR #5323)
+- Study progression performance improvement (PR #5887)
 
-#### Modules 
+#### Bug Fixes
+- Fix width of select elements that have no label (PR #5932)
+- Fix edge-case that gave a confusing error message (PR #5956)
+- Fix hidden examiner form field bug (PR #6085)
 
-##### Issue Tracker
-- The issue_tracker module now has the feature of uploading attachments to new or existing issues.
+### Modules 
 
-##### Battery Manager
- - New module created to manage the entries in the test_battery table of the database.
- This allows projects to modify their instrument battery without requiring backend access.
+#### Battery Manager
+- New module created to manage the entries in the test_battery table of the database.
+This allows projects to modify their instrument battery without requiring backend access.
  (PR #4221)
+ 
+#### Issue Tracker
+- The issue_tracker module now has the feature of uploading attachments to new or existing issues.
+ 
+#### Module Manager
+- New module created to manage the status of installed modules (PR #6015)
+
+#### Electrophysiology Browser
+- New module created to view electrophysiology data within LORIS. (PR #5230)
 
-##### Module Manager
- - New module created to manage the status of installed modules (PR #6015)
+#### Candidate Parameters
+- Add tab to view and edit a candidate's date of birth. (PR #4915)
+- Add date of death feature for LORIS. Store and calculate age based on a date of 
+death for candidates. (PR #4929)
 
-##### Electrophysiology Browser
- - New module created to view electrophysiology data within LORIS. (PR #5230)
+#### Create Timepoint
+- Add language parameter when creating a new timepoint for multilingual studies. (PR #4976)
+ 
+#### Data Release
+- UI improvements, bugfixes and addition of a filter for searching across release 
```

Comment:
I don't understand what this point means

Reply:
data release module was just a menu, now its a menu filter